packages/block-editor/CHANGELOG.md

```diff
@@ -6,9 +6,10 @@
 - Added the `addToGallery` property to the `MediaPlaceholder` component. The component passes the property to the `MediaUpload` component used inside the placeholder.
 - Added the `isAppender` property to the `MediaPlaceholder` component. The property changes the look of the placeholder to be adequate to scenarios where new files are added to an already existing set of files, e.g., adding files to a gallery.
 - Added the `dropZoneUIOnly` property to the `MediaPlaceholder` component. The property makes the `MediaPlaceholder` only render a dropzone without any other additional UI.
-- Added a cancel link to the list of buttons in the `MediaPlaceholder` component which appears if an `onCancel` handler exists
-- Added the usage of `mediaPreview` for the `Placeholder` component to the `MediaPlaceholder` component
-- Added a an `onDoubleClick` event handler to the `MediaPlaceholder` component
+- Added a cancel link to the list of buttons in the `MediaPlaceholder` component which appears if an `onCancel` handler exists.
+- Added the usage of `mediaPreview` for the `Placeholder` component to the `MediaPlaceholder` component.
+- Added a an `onDoubleClick` event handler to the `MediaPlaceholder` component.
+- Added a `forwardRef` to the `PlainText` component.
```

Comment:
This should be rephrased to better explain how `PlainText` was extended:

```suggestion
- Added a way to pass special `ref` property to the `PlainText` component.
```

I'm not 100% sure it reads well, but I hope it illustrates what I mean.